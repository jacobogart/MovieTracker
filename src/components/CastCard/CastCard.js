import React from 'react';

export default function CastCard(props) {
  const { name, headshot, character, id} = props.data;
  
  return (
    <div>
      <h2>{name}</h2>
      <h2>{character}</h2>
      <img className="headshot" src={headshot} alt={`${name} headshot`} />
    </div>
  );
}
