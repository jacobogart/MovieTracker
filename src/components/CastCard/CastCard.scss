.CastCard {
  background: white;
  height: 275px;
  margin: 0 15px 15px 0;
  width: 150px;

  .headshot {
    height: 200px;
    width: 150px;
    object-fit: cover;
  }

}