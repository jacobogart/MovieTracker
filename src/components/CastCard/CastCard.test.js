import { shallow } from 'enzyme';
import React from 'react';
import CastCard from './CastCard';

const mockCastMember = {
  name: 'Chris Evans',
  character: "Captain America",
  headshot: 'fsdfasdfa.png'
}

describe('CastCard', () => {
  it('should match the snapshot', () => {
    let wrapper = shallow(<CastCard data={mockCastMember}/>);
    expect(wrapper).toMatchSnapshot();
  });
});