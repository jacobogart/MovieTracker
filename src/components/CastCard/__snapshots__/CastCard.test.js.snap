// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CastCard should match the snapshot 1`] = `
<div>
  <h2>
    Chris Evans
  </h2>
  <h2>
    Captain America
  </h2>
  <img
    alt="Chris Evans headshot"
    className="headshot"
    src="fsdfasdfa.png"
  />
</div>
`;
